web-ui/src/main/resources/catalog/components/history/GnHistoryService.js

```diff
@@ -35,17 +35,22 @@
         '$http', '$filter',
       function($http, $filter) {
         this.delete = function (step) {
-          return $http.delete('../api/records/' + step.id.metadataId + '/status/' +
-            step.id.statusId + '.' + step.id.userId + '.' + step.id.changeDate.dateAndTime);
+          return $http.delete('../api/records/' + step.uuid + '/status/' +
```

Comment:
Not sure how this works with the workflow enabled and the draft metadata copies, as there're 2 metadata with different id, but same uuid.

Reply:
I have reverted this back to use the id.